google Oauth/app.py

```diff
@@ -0,0 +1,78 @@
+import config
+from flask import Flask, redirect, url_for, session
+from flask_sqlalchemy import SQLAlchemy
+from flask_oauth import OAuth
+# from authlib.flask.client import OAuth
+ 
+# You must configure these 3 values from Google APIs console
+# https://code.google.com/apis/console
+GOOGLE_CLIENT_ID = config.GOOGLE_API_CLIENT_ID
+GOOGLE_CLIENT_SECRET = config.GOOGLE_API_CLIENT_SECRET
+REDIRECT_URI = '/oauth2callback'  # one of the Redirect URIs from Google APIs console
+ 
+SECRET_KEY = config.SECRET_API_DEV_KEY
+DEBUG = True
+ 
+app = Flask(__name__)
+app.debug = DEBUG
+app.secret_key = SECRET_KEY
+oauth = OAuth()
+ 
+google = oauth.remote_app('google',
+base_url='https://www.google.com/accounts/',
+authorize_url='https://accounts.google.com/o/oauth2/auth',
+request_token_url=None,
+request_token_params={'scope': 'https://www.googleapis.com/auth/userinfo.email',
+'response_type': 'code'},
+access_token_url='https://accounts.google.com/o/oauth2/token',
+access_token_method='POST',
+access_token_params={'grant_type': 'authorization_code'},
+consumer_key=GOOGLE_CLIENT_ID,
+consumer_secret=GOOGLE_CLIENT_SECRET)
+ 
+@app.route('/')
```

Comment:
Good work bro! Is it possible we have all the methods in a separate file? I don't think it is appropriate for the app.py to be crowded as thus